src/test/scala/org/zouzias/spark/lucenerdd/LuceneRDDSpec.scala

```diff
@@ -122,6 +122,7 @@ class LuceneRDDSpec extends FlatSpec
     val maps = List(Map( "a" -> "hello"), Map("b" -> "world"), Map("c" -> "how are you"))
     val rdd = sc.parallelize(maps)
     luceneRDD = LuceneRDD(rdd)
+    luceneRDD.cache()
```

Comment:
Could you explain why is this needed?

Reply:
`.cache()` triggers the creation of the Lucene index.